store/tikv/gcworker/gc_worker.go

```diff
@@ -307,13 +314,22 @@ func RunGCJob(ctx goctx.Context, s tikv.Storage, safePoint uint64, identifier st
 	if err != nil {
 		return errors.Trace(err)
 	}
-	err = doGC(ctx, s, safePoint, identifier)
+	err = doGC(ctx, s, safePoint, identifier, gcDefaultConcurrency)
 	if err != nil {
 		return errors.Trace(err)
 	}
 	return nil
 }
 
+func (w *GCWorker) doGC(ctx goctx.Context, safePoint uint64) error {
+	gcConcurrency, err := w.loadGCConcurrencyWithDefault()
+	if err != nil {
+		log.Errorf("")
+		gcConcurrency = gcDefaultConcurrency
+	}
+
+	return doGC(ctx, w.store, safePoint, w.uuid, gcConcurrency)
```

Comment:
So many `doGC` makes me confused, could you change this function name to `doGCParallel`

Reply:
And this method can be merged to `runGCJob`

Reply:
ok

Reply:
Empty function？

Reply:
?this func is used to get the gcConcurrency